apps/www/components/Article/ArticleRecommendationsFeed.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useMemo } from 'react'
+import { Center, Interaction, fontStyles } from '@project-r/styleguide'
+import { useArticleRecommendationsQuery } from './graphql/getArticleRecommendations.graphql'
+import { css } from 'glamor'
+import Loader from '../Loader'
+import DocumentList from '../Feed/DocumentList'
+import { useTranslation } from '../../lib/withT'
+
+const styles = {
+  heading: css({
+    marginBottom: 15,
+    ...fontStyles.sansSerifMedium16,
+  }),
+  wrapper: css({
+    marginTop: 44,
+  }),
+}
+
+type ArticleSuggestionsFeedProps = {
+  path: string
+}
+
+const ArticleRecommendationsFeed = ({ path }: ArticleSuggestionsFeedProps) => {
+  const { t } = useTranslation()
+  const { data, loading } = useArticleRecommendationsQuery({
+    variables: {
+      path,
+    },
+  })
+
+  return (
+    <Center>
+      <Loader
+        loading={loading}
+        render={() => (
+          <>
+            {data.article.meta.recommendations?.nodes.length > 0 && (
```

Comment:
`data.article` will be null when the article is not found (e.g. path changed in the mean time). Also according to error logs `data` is sometimes `undefined`:
<img width="739" alt="Screenshot 2022-04-22 at 23 23 33" src="https://user-images.githubusercontent.com/410211/164796869-767be438-d817-49af-b774-071d5de13a1b.png">

Why are we doing a separate query instead of adding it to the article page query?

Reply:
It's a separate query since we don't want it to be included in the SSG version of the page.
The ArticleRecommendationsFeed is only rendered in case a user is authenticated and an active member.

Reply:
Thank you for the fix. Go for deploying it. I think data being undefined might be in case of error, I think we should display the error (fwd to loader) and move the heading outside the loader, but that's something for next week.